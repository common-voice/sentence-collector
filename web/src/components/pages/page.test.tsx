import React from 'react';
import { render } from '@testing-library/react';

import Header from '../header';
import Footer from '../footer';
import Page from './page';

jest.mock('../header');
jest.mock('../footer');

test('should render page', () => {
  (Header as jest.Mock).mockReturnValue(<div>...Header...</div>);
  (Footer as jest.Mock).mockReturnValue(<div>...Footer...</div>);
  const { container } = render(<Page />);
  expect(container).toMatchSnapshot();
});
