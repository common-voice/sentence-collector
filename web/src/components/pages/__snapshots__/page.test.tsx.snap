// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render page 1`] = `
<div>
  <div>
    ...Header...
  </div>
  <main />
  <div>
    ...Footer...
  </div>
</div>
`;
